source/blog/2018-05-11-the-emberjs-times-issue-46.md

```diff
@@ -62,41 +66,32 @@ Or if you feel in a particulary good mood, look through existing issues in the [
 
 ---
 
-## [Moar #EmberJS2018 blog posts wanted!](https://emberjs.com/blog/2018/05/02/ember-2018-roadmap-call-for-posts.html)
-
-We've already seen a lot of thoughtful blog posts from the Ember team's [call for community blog posts](https://emberjs.com/blog/2018/05/02/ember-2018-roadmap-call-for-posts.html). But we're hoping for more! Have something to say?
-
-Write a blog post by **May 30th** to propose goals and direction for Ember in the remainder of 2018. The content of these posts will help the core team to draft their first Roadmap RFC.
-
-Looking for inspiration? Check out the [**#EmberJS2018** hashtag](https://twitter.com/search?q=%23EmberJS2018) on Twitter or [@zinyado's repo collecting posts](https://github.com/zinyando/emberjs2018-posts) on GitHub.
-
----
-
-## [SOMETHING ELSE HERE](#your-url-here)
-
-
----
-
-
 ## [Contributors' Corner](https://guides.emberjs.com/v3.1.0/contributing/repositories/)
 
-Many thanks to  <a href="https://github.com/rwjblue" target="gh-user">@rwjblue</a>, <a href="https://github.com/krisselden" target="gh-user">@krisselden</a>, <a href="https://github.com/toddjordan" target="gh-user">@toddjordan</a>, <a href="https://github.com/Gaurav0" target="gh-user">@Gaurav0</a>, <a href="https://github.com/kennethlarsen" target="gh-user">@kennethlarsen</a>, <a href="https://github.com/runspired" target="gh-user">@runspired</a>, <a href="https://github.com/ryanto" target="gh-user">@ryanto</a> and <a href="https://github.com/nummi" target="gh-user">@nummi</a> for contributing to Ember core projects this week. 💖
+Many thanks to <a href="https://github.com/rwjblue" target="gh-user">@rwjblue</a>, <a href="https://github.com/krisselden" target="gh-user">@krisselden</a>, <a href="https://github.com/toddjordan" target="gh-user">@toddjordan</a>, <a href="https://github.com/kennethlarsen" target="gh-user">@kennethlarsen</a>, <a href="https://github.com/runspired" target="gh-user">@runspired</a>, <a href="https://github.com/ryanto" target="gh-user">@ryanto</a>, <a href="https://github.com/nummi" target="gh-user">@nummi</a>, <a href="https://github.com/chancancode" target="gh-user">@chancancode</a>, <a href="https://github.com/twokul" target="gh-user">@twokul</a>, <a href="https://github.com/kellyselden" target="gh-user">@kellyselden</a>, <a href="https://github.com/sivakumar-kailasam" target="gh-user">@sivakumar-kailasam</a>, <a href="https://github.com/kategengler" target="gh-user">@kategengler</a>, <a href="https://github.com/pzuraq" target="gh-user">@pzuraq</a>, <a href="https://github.com/dfreeman" target="gh-user">@dfreeman</a>, <a href="https://github.com/mansona" target="gh-user">@mansona</a> and <a href="https://github.com/Gaurav0" target="gh-user">@Gaurav0</a> for contributing to Ember core projects this week. 💖
```

Comment:
Ah sorry - I thought I was being helpful but I guess the contributors list I copied from `whats-new-in-emberland` was too short?

Reply:
Ah no, that was already great and that's also exactly what I did!  I think there were just some more pull requests merged in the meantime and I thought I updated it to reflect the new contributors as well

Reply:
👍oh nice, wow a lot was merged, cool!